java-examples/src/test/java/edu/practice/datastructure/doublylinkedlist/DoublyLinkedListTest.java

```diff
@@ -0,0 +1,18 @@
+package edu.practice.datastructure.doublylinkedlist;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class DoublyLinkedListTest {
+
+	@Test
+	void test() {
+		fail("Not yet implemented");
+	}
+	void DoublyLinkedListTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.